src/main/build/version.h

```diff
@@ -26,7 +26,7 @@
 #define FC_FIRMWARE_IDENTIFIER      "BTFL"
 #define FC_VERSION_MAJOR            4  // increment when a major release is made (big new feature, etc)
 #define FC_VERSION_MINOR            2  // increment when a minor release is made (small new feature, change etc)
-#define FC_VERSION_PATCH_LEVEL      9  // increment when a bug is fixed
+#define FC_VERSION_PATCH_LEVEL      10  // increment when a bug is fixed
```

Comment:
The comment is misaligned, but is not a problem, it will return to 1 when we start version 4.3...